locales/tools/manage

```diff
@@ -138,14 +138,31 @@ const writeJsonFromCsv = (locales, toPath, sourceLanguage) => {
     }
 
     const csvs = files.map(file => `${locales}/${file}`).filter(file => path.extname(file) === '.csv')
-    const byLanguageModules = fromPairs(csvs.map(csvFile => [path.basename(csvFile, '.csv'), loadModules(csvFile, 'target_language')]))
 
-    if (!csvs) {
+    if (isEmpty(csvs)) {
       throw  new Error('A minimum of one CSV is required in order to build a JSON!')
     }
 
+    const byLanguageModules = fromPairs(csvs.map(csvFile => [path.basename(csvFile, '.csv'), loadModules(csvFile, 'target_language')]))
+
     const sourceModules = loadModules(csvs[0], 'source_language')
 
+    // Show which source languages differ
+    csvs.forEach((csv, index) => {
+      if (!isEqual(loadModules(csv, 'source_language'), sourceModules)) {
+        console.log('source language 1', sourceModules)
+        console.log()
```

Comment:
why the space?

Reply:
That was done by @klinzo and is not part of this PR